ios-coding-guidelines.md

```diff
@@ -46,4 +46,14 @@ These guidelines are meant to be used without Interface Builder and teach you ho
 
 ## Xcode
 
-1. **No White-Spaces In Group/Folder Names** Scripts do not like whitespaces. Since we have a lot of automation by scripts, please use `_` as replacement for them.
\ No newline at end of file
+1. **No White-Spaces In Group/Folder Names** Scripts do not like whitespaces. Since we have a lot of automation by scripts, please use `_` as replacement for them.
+
+## Accessibility
+
+1. * Use the accessibility id of a view to make it identifiable for unit tests or voice over announcements. The syntax for those identifiers should be `aid.viewController.view` or if further scoping is needed `aid.viewController.parentView.view`.
+2. * Set the appropriate accessibility traits if not set automatically. While buttons usually have it inherently set, labels functioning as headers need need their trait set to `.header`. The same applies for search fields. In cases were another view serves as a button, it's trait should be set to `.button`.
+3. * Use post notifications to inform about screen changes, for example ride cancellation, disappearing or appearing popovers or warning labels.
```

Comment:
Here I would like to add that you should use them sparsely and only when really needed, since they interrupt/cancel VO messages triggered by user actions (see the switched in Account where the hint message cancels the message bout the switch which is currently in focus. It would be much better if the updated a11y label of the switch would contain the hint message instead of posting a notification that interrupts all other messages.)